### DISCUSSION TITLE
erro in solving LMIs by optimize

### DISCUSSION THREAD

**Discussion by MostaFaramin** [yalmip/YALMIP#1003] [General discussion] [answered]
Hi

Previously I had no problem in solving LMIs by optimize command. But now, when I run a simple code like the code below, I have the following error. How can I solve it?

clear
clc
close all

A = [-1 2;-3 -4];
P = sdpvar(2,2);
F = [P>=eye(2), A'*P+P*A <= -eye(2)];
optimize(F, []);
P0 = value(P);

**Error using optimize/optimize_globally (line 693)
When optimizing globally ([x0] is empty), both [lb] and [ub] must be defined.

Error in optimize (line 320)
        [sol, fval, exitflag, output] = optimize_globally(varargin{:});

Error in Test (line 8)
optimize(F, []);**

**Comment by johanlofberg** [comment, marked as answer]
you are calling some other function called optimize. that is not yalmips optimize

**Comment by MostaFaramin** [comment]
Thank you Johan

I've always used this function to solve LMI problems, but I've never had the same trouble. How can I solve this issue?

**Reply by johanlofberg** [reply]
Remove the file you have installed which gets in the way

**Reply by MostaFaramin** [reply]
Thank you very much for your prompt reply and a good hint. 
Recently, I installed a fractional system toolbox, and now when I removed that toolbox, everything was solved.
